actions/pdp-renderer/render.js

```diff
@@ -55,12 +62,17 @@ async function generateProductHtml(sku, urlKey, context) {
       const baseUrl = getProductUrl(baseProduct, context);
       if (Array.isArray(option.values)) {
         option.values = option.values.map((value) => ({
-          ...value,
```

Comment:
Was this intentional removal of the spread? Figured you would just append `title` prop after if that's all you need.